src/FakeItEasy/Analysis/MustUseReturnValueAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace FakeItEasy.Analysis
+{
+    using System;
+
+    /// <summary>
+    /// Indicates that the return value of a method should not be ignored.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public sealed class MustUseReturnValueAttribute : Attribute

```

Comment:
Has to be public to be seen by Roslyn (otherwise it's only accessible by reflection). It's unfortunate, because the user isn't supposed to see it... Perhaps the Roslyn assembly can be added as a friend assembly; not sure it would work, but it's worth trying.